media-gfx/darktable/darktable-2.4.0.ebuild

```diff
@@ -79,6 +75,9 @@ pkg_pretend() {
 }
 
 src_prepare() {
+	default
+	eapply "${FILESDIR}"/"${P}"-find-opencl-header.patch
```

Comment:
This is an EAPI 6 ebuild, so instead of `eapply` just put this in a global `PATCHES` array after the dependency declarations.